### DISCUSSION TITLE
Practica de html

### DISCUSSION THREAD

**Discussion by Blan25ga** [Launch-X-Latam/MisionFrontEnd#209] [Q&A]
Primero que nada hola a todos.
Tengo una consulta, en el video de ayer se hablo que al practica sobre "html" era sobre una taqueria, y en el repositorio dicta el listado de tareas a realizar un sitio sobre una pasteleria.
Tomamos el tema de pasteleria o el tema de taqueria o cualquier otro resto?? y depende el tema elegido seguimos como ejemplo a tener los puntos que estan en el repo?? (esto como guia)

**Comment by EsthebanDiego** [comment]
Tengo entendido que si no quisieras hacer la pastelería puedes usar la práctica que tu quisieras, aunque la práctica que hicieron en el video de ayer es más bien una guía para tu propio proyecto, ya sea pastelería o uno tuyo propio

**Reply by Blan25ga** [reply]
AAAH ok, tomamos de guia lo que dice en el repo sobre pasteleria pero realizamos nuestra propia web. siempre y cuando sea similar, por el tema de pedidos en cada punto solicitado en el repo.
Muchas gracias Esteban!!!
